src/expand.rs

```diff
@@ -75,6 +75,12 @@ impl Expander {
         }
     }
 
+    #[doc(hidden)]
+    #[deprecated(since = "0.3.6", note = "Use `escape` instead.")]
```

Comment:
The last version is 0.4.0, 0.3.6 doesn't exist:

```suggestion
    #[deprecated(since = "0.4.0", note = "Use `escape` instead.")]
```